02-oop-inheritance/topic-2.js

```diff
@@ -0,0 +1,166 @@
+$(document).ready(function(){
+	/*Define Movie*/
+var Movie = function () {
+  /*Set Vars*/
+  this.attributes = {};
+  this.attributes.actors = [];
+};
+
+
+
+ /*Define Movies Prototype as a Module*/ 
+Movie.prototype = (function () {
```

Comment:
Awch... this is not good :)
(function () { ... } ()); or (function () { ... })(); are IIFE's Immediately Invoked Function Expression, it means that they don't need to be invoked later after having been created. For example jQuery is and IIFE, when you load the library, you don't need to do anything to activate it. This said, your prototype should be an object, not and IIFE.